applications/nrf5340_audio/src/audio/streamctrl_unicast_server.c

```diff
@@ -456,6 +423,38 @@ static void le_audio_msg_sub_thread(void)
 	}
 }
 
+/**
+ * @brief	Create zbus subscriber threads.
+ *
+ * @return	0 for success, error otherwise.
+ */
+static int zbus_subscribers_create(void)
+{
+	int ret;
+
+	button_msg_sub_thread_id = k_thread_create(
+		&button_msg_sub_thread_data, button_msg_sub_thread_stack,
+		CONFIG_BUTTON_MSG_SUB_STACK_SIZE, (k_thread_entry_t)button_msg_sub_thread, NULL,
+		NULL, NULL, K_PRIO_PREEMPT(CONFIG_BUTTON_MSG_SUB_THREAD_PRIO), 0, K_NO_WAIT);
+	ret = k_thread_name_set(button_msg_sub_thread_id, "BUTTON_MSG_SUB");
```

Comment:
Suggest we rename:
```suggestion
	ret = k_thread_name_set(button_msg_sub_thread_id, "BUTTON_ZBUS_SUB");
``` : below as well